### PR TITLE
Update Okteto CLI to `2.23.1`

### PR DESCRIPTION
**Pull request by andreafalzetti**

### CODE REVIEW

**Review by jLopezbarb — APPROVED** [review]

**Review by AdrianPedriza — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## 0.4.12
+- Update to Okteto CLI 2.23.1
```
Comment (viddo) [inline CHANGELOG.md:4]: Nit; clarify change scope, WDYT?
```suggestion
- Update minimum required Okteto CLI version to 2.23.1
```

Comment (andreafalzetti) [reply CHANGELOG.md:4]: Why `minimum required`? 🤔

**Review by mnevadom — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (viddo) [reply CHANGELOG.md:4]: Based on [prev earlier changelog entries and var name](https://github.com/search?q=repo%3Aokteto%2Fremote-kubernetes%20minimum&type=code)? Not too familiar with ramifications of the specified version, but if one installs a newer minor/patch version I'd expect the extension to still work, wouldn't it? e.g. a future v2.24x or such I mean.

Comment (andreafalzetti) [reply CHANGELOG.md:4]: ah ok, thanks @viddo for clarifying, it's my first contribution to this repo and I didn't investigate much!

Comment (andreafalzetti) [reply CHANGELOG.md]: I have used the same copy as of other releases (see release notes) but I will use this suggestion 👍

**Review by rberrelleza — APPROVED** [review]

**Pull request merged**